server/the-usher-openapi-spec.yaml

```diff
@@ -428,6 +428,33 @@ paths:
         404:
           $ref: '#/components/responses/NotFound'
 
+  /roles/{role_key}/permissions:
+    get:
+      'x-swagger-router-controller': 'roles/permissions'
+      operationId: getRolesPermissions
+      summary: "Roles: Get list of Permissions"
+      parameters:
+        - $ref: '#/components/parameters/roleKeyPathParam'
+      tags:
+        - Admin APIs
+      security:
+        - bearerAdminAuth: []
+      responses:
+        200:
+          description: Returns a list of permissions for the subject roles
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: "#/components/schemas/PermissionObject"
+        401:
```

Comment:
@k1ch do you want to add the 400 error code response since we have a test for it.